### ISSUE TITLE
Toaru Majutsu no Virtual-On [PCSG01089]

### ISSUE THREAD

**Issue by 2907405506** [Vita3K/compatibility#506]
# App summary
- App name: Toaru Majutsu no Virtual-On [PCSG01089]
- App serial: [PCSG01089]
- App version: v1.02

# Vita3K summary
- Version: v0.1.3 2227
- Build number: 26351918

# Test environment summary
- Tested by: xiaofeigami(2907405506)
- OS: Windows 10
- CPU: Intel core2 duo E8500
- GPU: NVIDIA GTX750
- RAM: 4 GB

# Issues

# Screenshots
![QQ截图20210611200239](https://user-images.githubusercontent.com/50038528/121797517-3fd1d480-cc53-11eb-94b3-cce6e390757d.png)
![屏幕截图 2021-06-13 140634](https://user-images.githubusercontent.com/50038528/121797524-47917900-cc53-11eb-850e-d4de409a73cc.png)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/6643383/vita3k.log)

# Recommended labels

- audio bug
- color bug
- Bootable

**Comment by 2907405506** [comment]
The game can enter normally

The tone of the audio is too high

Inaccurate video color

Figure picture white

Fight scene flashback

**Comment by 2907405506** [comment]
The latest simulator will automatically exit when running this game(v0.1.3 2269),
![NIEPCT3@MJW0)3V_HI( )RI](https://user-images.githubusercontent.com/50038528/134156505-1c22dfc5-6af9-4aef-b2ab-ca30f8e1f377.png)
I found that in the historical version, the game works normally.
The last version that can run the game is vita3k v0.1.3 2247-c0697d1
![1$H0(T3HHQG7(69}E42N`%3](https://user-images.githubusercontent.com/50038528/134156259-c05d2e1f-affa-4a32-bd85-7b6aa0cc05c0.png)
But the character voice disappeared

**Comment by Zangetsu38** [comment]
yes, issue is already know, probleme for now, fix this issue broken again two other game

**Comment by GamingDucking** [comment]
as of https://github.com/Vita3K/Vita3K/commit/e58419c001c8809b70e30243398fd3eaec0f64dc, the game is playable under OpenGL. On Vulkan, it simply crashed without booting.
# Screenshots
![1](https://github.com/Vita3K/compatibility/assets/90236481/7cbc9002-4449-4e47-8bcf-a296f9e27ba6)
![2](https://github.com/Vita3K/compatibility/assets/90236481/e0bb1582-913f-4e1d-acbf-325e8a821407)
![3](https://github.com/Vita3K/compatibility/assets/90236481/70c1eb2d-4b41-4409-b40c-04cd3a12243c)
![4](https://github.com/Vita3K/compatibility/assets/90236481/463a1b0a-38b3-4ad8-8ee5-9765230f71b3)
![5](https://github.com/Vita3K/compatibility/assets/90236481/5024edaa-28e9-4acd-ad7c-20b4f0631de1)
![6](https://github.com/Vita3K/compatibility/assets/90236481/bf30bf36-f1fd-4f0b-ab65-5d4faf54ac93)
# Logs
[vita3k (OGL).log](https://github.com/Vita3K/compatibility/files/11788215/vita3k.OGL.log)
[vita3k (Vulkan).log](https://github.com/Vita3K/compatibility/files/11788217/vita3k.Vulkan.log)
# Recommended Labels
- Playable
- vulkan crash

**Comment by github-user1112** [comment]
# Vita3K summary
Version: v0.2.1
Build number: 3967
Commit hash: https://github.com/Vita3K/commit/9916d86
GPU backend: Vulkan

# Test environment summary
Tested by: gtihub-user
OS: Linux Mint 22.3
CPU: AMD Ryzen 5 5600
GPU: Radeon Rx 6600
RAM: 16 GB

# Issues
Vulkan no longer crashes

# Screenshots

<img width="960" height="1042" alt="Image" src="https://github.com/user-attachments/assets/afb51364-5048-44d5-80d4-cfdeb27efda4" />

<img width="960" height="1042" alt="Image" src="https://github.com/user-attachments/assets/28ffafab-fe28-42cc-97b9-568427aca45c" />

# Log

# Recommended labels

**Comment by MakutaMaster962** [comment]
### Vita3k Summary
- Version: v0.2.1
- Build:  4058-6063154f
- Graphics backend: Vulkan

### Test environment summary
- Tested by: MakutaMaster962
- OS: Windows 10 Enterprise LTSC 21H2
- CPU: AMD Ryzen 7 9800X3D 8-Core Processor
- GPU: AMD Radeon RX 9060 XT
- RAM: 64 GB

### Issues
Emulator crashes whenever the game's "Attract Mode" (which starts after displaying the title screen for 10 or so seconds) starts. After the "Versus Screen" for the game's "Attract Mode" fades to white, then both the game & emulator crash. All other modes seem to work just fine.

### Screenshots

The "Versus Screen" just before the fade-to-white & then the crash:
<img width="962" height="576" alt="Image" src="https://github.com/user-attachments/assets/4d6be8dc-8554-420f-afa5-c501d32e57fc" />

### Log

[vita3k.log](https://github.com/user-attachments/files/29719491/vita3k.log)

### Recommended labels
- vulkan issue

**Comment by github-user1112** [comment]
my bad. should have tested the modes as well
